appveyor.yml

```diff
@@ -4,20 +4,36 @@ notifications:
 
 version: "{build}"
 clone_depth: 10
+configuration:
+  - BuildCheck
+  - ReleasePackage
```

Comment:
Can we unify them?
Can we build and archive package in each job like we did in Groonga?

Reply:
Yes. I've fixed.